invenio_app/__init__.py

```diff
@@ -143,6 +144,12 @@
 ``<sys.prefix>/var/instance/`` where <sys.prefix> is your Python root prefix
 (e.g. ``/usr/``)
 
+Securing your instance
+~~~~~~~~~~~~~~~~~~~~~~~~~
+If you have a proxy in front of your invenio-app instance you need to specify
```

Comment:
If I may suggest, it might be more useful to mention more the config `WSGI_PROXIES` that is exposed (please feel free to improve English and readability here!):
```
The Invenio documentation <link> explains in details how to secure an Invenio instance. It is important to note that if you deploy your Invenio instance with at least one reverse proxy in front, then you will have to set the configuration variable `WSGI_PROXIES` accordingly to correctly handle the `X-Forwarded-*` headers.
```
```

Reply:
Definitely your version sounds clearer- I will use it.